botx/bots/mixins/requests/chats.py

```diff
@@ -188,3 +190,37 @@ async def add_admin_roles(
             AddAdminRole(group_chat_id=chat_id, user_huids=user_huids),
             credentials=credentials,
         )
+
+    async def pin_message(
+        self: BotXMethodCallProtocol,
+        credentials: SendingCredentials,
+        chat_id: UUID,
+        sync_id: UUID,
```

Comment:
credentials has chat(and message) id, maybe use them as default?

Reply:
Custom credentials can be passed which may not have chat and sync IDs as these fields are optional.